packages/target-chrome-core/package.json

```
@@ -21,7 +21,9 @@
   "main": "src/index.js",
   "dependencies": {
     "@loki/browser": "^0.25.1",
-    "@loki/core": "^0.25.0"
+    "@loki/core": "^0.25.0",
+    "@loki/integration-core": "^0.25.0",
+    "debug": "^4.2.0"
```

```suggestion
    "debug": "^4.1.1"
```
This way we only have one version of this dependency defined in the workspace. If we're gonna bump, I'd argue we bump all packages.